### PR TITLE
fix: Fix liveness test

### PR DESCRIPTION
**Pull request by mattverse**
## Description

Overrides liveness test fix followed by previous liveness test fix https://github.com/osmosis-labs/cosmos-sdk/pull/54.
Basically github action was failing due to different versions between cosmos-sdk and our fork, thus docker being able to not find certain directories in the dockerFile.

This PR introduces changes in dockerFile so that it matches our file directory, consequently fixing current bug in liveness test.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by mattverse** [comment]
Tested in local environment, git actions on liveness tests would not run on this PR.

File: Makefile [hunk 1]
```diff
@@ -480,7 +480,7 @@ proto-update-deps:
 ###                                Localnet                                 ###
 ###############################################################################
 
-# Run a 4-node testnet locally
+# Run a 4-node testnet locally via docker compose
```
Comment (ValarDragon) [inline Makefile:483]: Only change appears to be a comment change? Still want this merged?

Comment (mattverse) [reply Makefile:483]: So sorry. Must have been a confusion while handling git. 
New commit has been made with the change included.

**Comment by mattverse** [comment]
Basically our liveness-test was failing because the dockerfile was trying to copy the `db` directory, which is included in cosmos-sdk, but not our fork
```
Step 5/20 : COPY db/go.mod db/go.sum /work/db/
COPY failed: file not found in build context or excluded by .dockerignore: stat db/go.mod: file does not exist
```

**Review by ValarDragon — APPROVED** [review]

**Comment by ValarDragon** [comment]
Can you fix the merge conflict, not sure whats the right thing here

**Pull request merged**